Automattic/jetpack
# Voice to Content: register block just for internal P2s

Part of the release process.

## Proposed changes:

* Check if the site is an internal P2 before registering the block, so only internal P2s can use it for now
* Move the block from beta to production, so users can access it without proxied connections, allowing easy usage from mobile devices

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* We need to test it on internal P2s, other P2s, Simple sites and Jetpack sites

---

Testing on the WordPress.com environment:

* First, apply this branch to your sandbox using the `jetpack-download` command provided below or using `jetpack rsync`
* Then, sandbox a site from each type:
   * an internal, A8C P2 site (our team's P2, for example)
   * another P2 that is not internal, can be your usual test P2
   * a Simple site
* After it, go to the block editor on each of the sites and try to insert a Voice to Content block
   * The block should **not** be available on the Simple site
   * The block should **not** be available on the test P2 site
   * The block should **be** available on the internal, A8C P2 site, and it should not be flagged as beta

---

Testing on the Jetpack site environment:

* Spin a new JN site with this branch
* Connect it and go to the block editor
* Try to insert a Voice to Content block; it should not be available, even when switching to the beta blocks variation
